Makefile

```diff
@@ -47,13 +47,7 @@ endif
 
 ifeq (,$(BUILD_TOOL))
   ifeq (Ninja,$(CMAKE_GENERATOR))
-    ifneq ($(shell $(CMAKE_PRG) --help 2>/dev/null | grep Ninja),)
-      BUILD_TOOL = ninja
-    else
-      # User's version of CMake doesn't support Ninja
-      BUILD_TOOL = $(MAKE)
-      CMAKE_GENERATOR := Unix Makefiles
-    endif
+    BUILD_TOOL = ninja
```

Comment:
why remove this fallback?

Reply:
It's dead code. It's only useful if user's version of cmake doesn't support ninja, but our minimum required cmake version is 3.10, which supports ninja.

Reply:
This fallback was for users that don't have `ninja` installed.

Reply:
I built neovim without ninja installed, it defaults to make (which I forgot how slow it was, dear lord).